src/Builder/Printer/ClassPrinter.php

```diff
@@ -17,6 +17,34 @@
 
 class ClassPrinter extends Standard
 {
+    /**
+     * Determine whether a list of nodes uses multiline formatting.
+     *
+     * @param (\PhpParser\Node|null)[] $nodes Node list
+     *
+     * @return bool Whether multiline formatting is used
+     */
+    protected function isMultiline(array $nodes): bool
+    {
+        if (!$nodes) {
+            return false;
+        }
+
+        if (count($nodes) === 1 && current($nodes) !== null) {
+            $node = current($nodes);
+            $startPos = $node->getStartTokenPos() - 1;
+            $endPos = $node->getEndTokenPos() + 1;
+            $text = $this->origTokens->getTokenCode($startPos, $endPos, 0);
+            if (strpos($text, "\n") === false) {
+                return false;
+            }
+
+            return true;
```

Comment:
Instead of 

```
if (strpos($text, "\n") === false) {
                return false;
            }

            return true;
```

you could use 

```
return strpos($text, "\n") !== false;
```